### PR TITLE
Adding Find and Click-on Steps

### PR DESCRIPTION
**Pull request by OdeclasV**
Senator Schatz's form was not properly selecting the "Share your opinion" option in the form's dropdown menu. Sometimes it would select and others it would fail to do so. I tweaked that part of the yaml a little bit to look for and click the specific element in the that menu. I also added a wait time at the beginning and decreased one that was previously there by 2s.

### CODE REVIEW

File: members/S001194.yaml [hunk 1]
```diff
@@ -2,21 +2,21 @@ bioguide: S001194
 contact_form:
   method: post
   action: /contact
+  usePhantomJsLogging: true
```
Comment (tomdionesotes) [inline members/S001194.yaml]: why do we have to force phantom?

Comment (OdeclasV) [reply members/S001194.yaml]: I think we should as I noticed the dropdown menu uses JS. 

See the `onchange` jquery-esque value in the `select id="actions"` element in the image below. 

image

I could be wrong, but wanted to play it safe with this change hence why I added phantom. 

@tomdionesotes

Comment (tomdionesotes) [reply members/S001194.yaml]: The form uses JS to fetch the rest of the inputs etc once we select a value in `select#actions`, sure. Lots of other forms work like this, and AFAIK their yamls work fine without forcing phantom.

Once we select a value in `select#actions` we can check if the rest of the elements are loaded with wait / find - unless the find is returning a false positive. 

You can test for a find false positive by temporarily editing this yaml to remove the `select#actions` click step, add a find for an input like first name, and see if the find succeeds in the tester.

Comment (OdeclasV) [reply members/S001194.yaml]: Oh ok got it. I did the test you suggested and it failed to find `firstname`, so at least now I know that other elements were not loading before a element in `select#actions` was selected. 

I also tested the changes w/o phantom and they passed. Taking phantom out, are you OK with the `find` and `click-on` changes in this yaml? 

@tomdionesotes

**Pull request merged**